addons/web_editor/static/src/js/wysiwyg/widgets/image_crop_widget.js

```diff
@@ -92,8 +92,9 @@ const ImageCropWidget = Widget.extend({
         this._onDocumentKeydown = this._onDocumentKeydown.bind(this);
         // We use capture so that the handler is called before other editor handlers
         // like save, such that we can restore the src before a save.
-        this.document.addEventListener('mousedown', this._onDocumentMousedown, {capture: true});
-        this.document.addEventListener('keydown', this._onDocumentKeydown, {capture: true});
+        this.ownerDocument = this.$el[0].ownerDocument;
+        this.ownerDocument.addEventListener('mousedown', this._onDocumentMousedown, {capture: true});
```

Comment:
So this would become:

```suggestion
        this.el.ownerDocument.addEventListener('mousedown', this._onDocumentMousedown, {capture: true});
```